templates/make.mpd

```diff
@@ -37,7 +38,7 @@ CXX           = <%cxx%>
 <%if(ld)%>
 LD            = <%ld%>
 <%else%>
-LD            = $(CXX) $(CCFLAGS) $(CPPFLAGS)
+LD            = $(CXX) $(CCFLAGS) $(PICFLAGS) $(GENFLAGS)
```

Comment:
Could this be a problem for projects that need something in CPPFLAGS to show up on the linker command line?

Reply:
I recall that this change was necessary for us as we used some linker(s) that barfed on -I and/or -D (which are for sure in CPPFLAGS). The change worked for us but now I look closely it isn't taking into account the possibility that e.g. compile_flags has something linker relevent set as an override (it's fixed empty otherwise) or somesuch. Since I can't even remember which toolchain it was had a problem with the status quo I think this is a bad idea and I should wthdraw this part. Sorry - and thanks for looking at it!